tokenize_rt.py

```diff
@@ -26,7 +29,7 @@ class Offset(collections.namedtuple('Offset', ('line', 'utf8_byte_offset'))):
     __slots__ = ()
 
     def __new__(cls, line=None, utf8_byte_offset=None):
-        # type: (Optional[int], Optional[int]) -> None
+        # type: (Type['Offset'], Optional[int], Optional[int]) -> 'Offset'
```

Comment:
do you need the first argument? it should be inferred for you

Reply:
IntelliJ gives the wrong hint otherwise (3rd screenshot). I can remove it and just followup with IntelliJ

Reply:
ah yeah, pycharm has their own type checker that probably doesn't understand `__new__` (if I had to guess)

Reply:
OK removed it. PR should be green now 🤞